### ISSUE TITLE
Move cursor to start of line on cut line

### ISSUE THREAD

**Issue by amcasey** [dotnet/roslyn#5462]
Cursor at ```|```; no selection.  Ctrl-x to cut.
```
> 11111|
```

Expected
```
> |
```

Actual (Note: space is virtual)
```
>      |
```

**Comment by amcasey** [comment]
Scenario 2:
```
> 1
. 11111|
```

Expected
```
> 1
. |
```

Actual (Note: space is virtual)
```
> 1
.      |
```

Now ctrl-v to paste the line back:

Expected
```
> 1
. 11111|
```

Actual
```
> 1
.      11111|
```

**Comment by genlu** [comment]
This will be fixed as part of https://github.com/dotnet/roslyn/pull/5472/files

**Comment by ManishJayaswal** [comment]
I believe that this is fixed by #5472
